pages/featured/index.js

```diff
@@ -0,0 +1,132 @@
+import config from "../../config";
```

Comment:
As the Tags and featured blogs' design is the same, you can create one component and reuse it for both tags and featured blogs